vmm_tests/vmm_tests/tests/tests/x86_64/openhcl_linux_direct.rs

```diff
@@ -396,6 +418,91 @@ async fn openhcl_linux_storvsp_dvd(config: PetriVmConfigOpenVmm) -> Result<(), a
     Ok(())
 }
 
+/// Test an OpenHCL Linux direct VM with a SCSI DVD assigned to VTL2, using NVMe
+/// backing, and vmbus relay. This should expose a DVD to VTL0 via vmbus.
+#[openvmm_test(openhcl_linux_direct_x64)]
+async fn openhcl_linux_storvsp_dvd_nvme(config: PetriVmConfigOpenVmm) -> Result<(), anyhow::Error> {
+    const NVME_INSTANCE: Guid = guid::guid!("dce4ebad-182f-46c0-8d30-8446c1c62ab3");
+    let vtl2_nsid = 1;
+    let nvme_disk_sectors: u64 = 0x4000;
+    let sector_size = 4096;
+
+    let vtl2_lun = 5;
+    let scsi_instance = Guid::new_random();
+
+    let mut vtl2_settings = None;
+
+    let disk_len = nvme_disk_sectors * sector_size;
+    let mut backing_file = tempfile::tempfile()?;
+    let mut bytes = vec![0_u8; disk_len as usize];
+    for i in 0..(disk_len / 0xff) {
+        // This should always be divisible by 64
```

Comment:
You could also write this code as something like the following, which would be more concise and avoid the pointer arith.

```rust
let data_chunk:[u8] = [0..63];
let mut bytes = vec![0_u8; disk_len as usize];
bytes.chunks_exact_mut(64).for_each(|v| { v.copy_from_slice(&data_chunk); }}
```